README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
Fix, not maintainable.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
Looks good, maintainable.

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Maintainable

Reply:
Test

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Efficient, Not maintainable